src/gameplay/enums/roomCreationType.ts

```diff
@@ -0,0 +1,6 @@
+export enum roomCreationTypeEnum {
+    RANKED_QUEUE = "RANKED_QUEUE",
+    UNRANKED_QUEUE = "UNRANKED_QUEUE",
+    CUSTOM_ROOM = "CUSTOM_ROOM",
+    INVALID_ROOM = "INVALID_ROOM"
```

Comment:
Do we need this?

Reply:
Yeah, you recommended creating the enum to track roomCreationType, right?
<img width="301" alt="image" src="https://user-images.githubusercontent.com/59691527/234180948-2262239d-6ac2-4f14-8ee4-c18e1384b981.png">

Reply:
I meant the Invalid room one lol

Reply:
Ah, I kept it in case of error, but yeah, I'll throw an error instead